lib/shelley/src/Cardano/Wallet/Shelley/Launch/Cluster.hs

```diff
@@ -403,10 +405,10 @@ poolConfigsFromEnv = isEnvSet "NO_POOLS" <&> \case
     False -> defaultPoolConfigs
     True -> []
 
-localClusterConfigFromEnv :: Maybe ClusterEra -> IO LocalClusterConfig
-localClusterConfigFromEnv defaultEra = LocalClusterConfig
+localClusterConfigFromEnv :: IO LocalClusterConfig
+localClusterConfigFromEnv = LocalClusterConfig
     <$> poolConfigsFromEnv
-    <*> clusterEraFromEnv defaultEra
```

Comment:
I think this was intended for running the integration tests on multiple eras on the same run.

Now I want to retrieve the `clusterEra` independently.

Arguably, a `localClusterConfigFromEnv :: ClusterEra -> IO LocalClusterConfig` that doesn't use `clusterEraFromEnv` internally might be neater in that case, but 🤷‍♂️